dnas/presence/zomes/coordinator/room/src/all_agents.rs

```diff
@@ -6,7 +6,7 @@ pub fn get_all_agents(_: ()) -> ExternResult<Vec<AgentPubKey>> {
     let path = Path::from(ALL_AGENTS);
     let links = get_links(
         LinkQuery::try_new(path.path_entry_hash()?, LinkTypes::AllAgents)?,
-        GetStrategy::Network,
+        GetStrategy::Local,
```

Comment:
Since the zome code gets changed anyway it might be nice to switch to the pattern that local/network can be specified in the zome call paramter? That way we can improve the UI code later to still go to the network in some cases, without having to change the zome code and dna hash.